source/support.tex

```diff
@@ -4283,9 +4283,10 @@
 \pnum
 The \tcode{partial_ordering} type is typically used
 as the result type of a three-way comparison operator\iref{expr.spaceship}
-that (a) admits all of the six two-way comparison operators (\ref{expr.rel}, \ref{expr.eq}),
-(b) does not imply substitutability,
-and (c) permits two values to be incomparable.%
+for a type that is intended to support
+all of the six two-way comparison operators (\ref{expr.rel}, \ref{expr.eq}),
```

Comment:
I think the space before `(\ref` should normally be a `~`.

Reply:
Fixed.